roles/ws_health_exporter/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+
+- block:
+
+  - name: ws_health_exporter | remove OLD exporter file
+    ansible.builtin.file:
+      path: "{{ _ws_health_exporter_old_file }}"
+      state: absent
```

Comment:
It is better to remove this step, this is "new" role and git history is empty,  this step may confuse new users.